Documentation/help_files/medical_dashboard_help.php

```diff
@@ -0,0 +1,235 @@
+                                        
+                    <p><?php echo xlt("Once a rule is setup it can be enabled for a particular patient");?>.
+                    
+                    <p><?php echo xlt("Upon reaching a predetermined point, either a date or value, the rule will trigger one or more events");?>:
+                        <ul>
+                            <li><?php echo xlt("Active Alert - that presents as a pop-up notification when a patient's chart is entered"); ?></li>
+                            <li><?php echo xlt("Passive Alert - that will be displayed in the Clinical Reminders widget section"); ?></li>
+                            <li><?php echo xlt("Patient Reminder - that is used to communicate relevant information pertaining to that particular Clinical Decision Rule and is shown in the Patient Reminders widget as Well as under Administration > Patient Reminders"); ?></li>
+                         </ul>
+                    
+                    <p><?php echo xlt("More information about Clinical Decision Rule can be found here");?>.
+                    <a href="https://www.open-emr.org/wiki/index.php/CDR_User_Manual" target="_blank"><i class="fa fa-external-link text-primary" aria-hidden="true" data-original-title="" title=""></i></a>
+                    
+                    <p><?php echo xlt("Appointments - shows all future appointments as well as Recalls");?>.
+                    
+                    <p><?php echo xlt("Recurrent Appointments - shows all recurring appointments");?>.
+                    
+                    <p><?php echo xlt("Past Appointments - will show all past appointments");?>.
+                    
+                    <p><?php echo xlt("Medical Problems - will show the patient's medical issues, Issues > Medical Problems");?>.
+                    
+                    <p><?php echo xlt("Allergies - will show the allergies listed under Issues > Allergies. If eRx is enabled the allergy list has to be entered on the eRx page");?>.
+                    
+                    <p><?php echo xlt("Medications - lists the medications under Issues > Medications. If eRx is enabled the medication list has to be entered on the eRx page");?>.
+                    
+                    <p><?php echo xlt("Immunizations - lists immunization history and allows for adding new entries or editing existing ones");?>.
+                    
+                    <p><?php echo xlt("Prescriptipn - lists the prescriptions of the current patient");?>.
+                    
+                    <p><?php echo xlt("Tracks - if the Track Anything feature is enabled the will display a list of values that can be tracked and graphed");?>.
+                    <a href="https://www.open-emr.org/wiki/index.php/Track_Anything_Form" target="_blank"><i class="fa fa-external-link text-primary" aria-hidden="true" data-original-title="" title=""></i></a>
```

Comment:
oh my goodness, this are is really nice documentation!!

Reply:
Tx